README.md

```diff
@@ -44,6 +44,14 @@ $ jsonschema2md -d examples/schemas -o examples/docs -v 04
 $ jsonschema2md -d examples/schemas -o examples/docs -v 06
 ```
 
+## Disabel header template
+In some cases you do not need a header cause it did not provide any usefull information. With the header parameter you could disabel the header template for the generation.
```

Comment:
```suggestion
In some cases you do not need a header because it does not provide any useful information. With the `--header` (or `-h`) parameter you can disable the inclusion of headers.
```